addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -348,6 +367,22 @@ options.Class.include({
         const color = style.getPropertyValue('--' + colorName).trim();
         return ColorpickerDialog.normalizeCSSColor(color);
     },
+    /**
+     * @private
+     * @param {JQueryElement} $target
+     * @returns {array} [shadowColor, shadowX, shadowY, shadowBlur, shadowSpread, shadowInset || undefined]
+     */
+    _getShadowValues($target) {
+        // shadow format: "rgb(0, 0, 255) 30px 0px 18px 5px" || "rgb(135, 90, 123) 2px 0px 0px 0px inset" || "none"
+        const values = $target.css('box-shadow');
+        if (values === "none") {
+            return ['black', 0, 0, 0, 0];
+        }
+        // should return shadow.split(/(?<!,)\s+/g) once ES8 is supported. (negative lookbehind regex).
+        const colorEndIndex = values.indexOf(')') + 1;
+        return [values.slice(0, colorEndIndex), ...values.slice(colorEndIndex).split(' ')]
+            .filter(str => str.length);
```

Comment:
Using the shadow class, I don't think you will need to split those values at all... then you just need to improve the split in we-multi.